packages/locales/src/locales/en.json

```diff
@@ -75,6 +75,9 @@
   "mcfunction.parser.entity-selector.player-name.too-long": "Player names cannot be longer than %0% characters",
   "mcfunction.parser.eoc-unexpected": "Expected more arguments",
   "mcfunction.parser.leading-slash": "a leading slash %0%",
+  "mcfunction.parser.macro.at-least-one": "at least one macro argument",
+  "mcfunction.parser.macro.illegal": "Illegal key character: “%0%”",
+  "mcfunction.parser.macro.key": "a macro key",
```

Comment:
Inconsistent use of "argument" and "key"

Reply:
Well the latter two errors are related to the contents within the parenthesis, consistently referred to as the key, and the former is related to the lack of any macro variable or argument, which pertains to the discussion we had as to what that is called but it is different from the consistently defined key.

Reply:
Would `invalid macro argument` or `malformed macro argument` describe this error message?

Reply:
Yeah, if we have consensus on a preferred wording, it makes sense to change now.